docker_mcp/models/params.py

```
@@ -0,0 +1,215 @@
+        default="", 
+        description="Host identifier"
+    )
+    stack_name: str = Field(
+        default="", 
+        description="Stack name"
+    )
+    compose_content: str = Field(
+        default="", 
+        description="Docker Compose file content"
+    )
+    environment: Dict[str, str] = Field(
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)